Ekh2o/front
# add toaster to signin and signup

#### Before submitting, make sure:
- [x] You have rebased your branch
- [x] You describe your changes
- [x] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

#### Describe below your changes:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/74325439/155143081-0b8e2cbf-b814-4454-9afd-a3d8fee1d661.png">
Add toaster when the Password or email are wrong
add toaster when the email is already exist

##### Describe below How to test:
LAUNCH YOUR BFF BEFORE TESTING
- Go to my account try to sign up with an email already existing.
for exemple antonin.val@gmail.com.
You will see the alert toaster .
- Go to my account try to loggin with random email and Password.
You will see the alert toaster.
- go to my account and try to loggin with antonin.val@gmail.com Password antonin
You will see your account
